src/helpers/filterInputPost.helper.ts

```diff
@@ -0,0 +1,36 @@
+import { POST } from "@/constants";
+import { IInputPost } from "@/interfaces";
+
+const filterInputPost = (post: IInputPost) => {
+  const {
+    id,
+    title,
+    content,
+    prUrl,
+    isFinished,
+    startTime,
+    endTime,
+    field,
+    lostThingImage,
+    place,
+    keepingPlace,
+    category,
+  } = post;
+
+  const defaultPost = { id, title, content, category };
+
+  if (category === POST.COMMON || category === POST.NOTICE) return defaultPost;
+
+  if (category === POST.CODE_REVIEW)
+    return { ...defaultPost, prUrl, isFinished };
+
+  if (category === POST.PROJECT)
+    return { ...defaultPost, startTime, endTime, field };
+
+  if (category === POST.LOST) return { ...defaultPost, lostThingImage, place };
+
+  if (category === POST.FOUND)
+    return { ...defaultPost, lostThingImage, place, keepingPlace };
```

Comment:
```suggestion
 switch (category) {
  case POST.COMMON:
  case POST.NOTICE:
    post = defaultPost;
    break;
  case POST.CODE_REVIEW:
    return { ...defaultPost, prUrl, isFinished };
  case POST.PROJECT:
    return { ...defaultPost, startTime, endTime, field };
  case POST.LOST:
    return { ...defaultPost, lostThingImage, place };
  case POST.FOUND:
    return { ...defaultPost, lostThingImage, place, keepingPlace };
}
```

switch case가 더 가독성이 좋을것같아유

Reply:
죠씁니다 변경완료했습닝당